modules/router-store/data-persistence/index.ts

```diff
@@ -0,0 +1 @@
+export * from './src/operators';
```

Comment:
This will also export types and interfaces from the `operators.ts` file. It would be better to only export operators like in the Nx repo: https://github.com/nrwl/nx/blob/master/packages/angular/index.ts

Reply:
Good catche